spec/state_machines/sar_permitted_events_spec.rb

```diff
@@ -0,0 +1,195 @@
+require 'rails_helper'
+
+describe 'SAR permitted events' do
```

Comment:
This should specify what method is being is being tested `ConfigurableStateMachine::permitted_events`.